examples/ma_transactions_example.py

```diff
@@ -0,0 +1,203 @@
+import asyncio
+from flow_py_sdk.cadence import address
+from flow_py_sdk.tx import Tx
+import logging
+from examples.common import Example, Config
+from flow_py_sdk.account_key import AccountKey
+from flow_py_sdk import ProposalKey
+from flow_py_sdk import flow_client
+from examples.common.utils import random_account
+from flow_py_sdk import cadence
+
+# -------------------------------------------------------------------------
+# Rretrieve a transaction by ID Class
+# -------------------------------------------------------------------------
+class GetTransactionByIdExample(Example):
```

Comment:
maybe this test would be clearer if a simple transaction was sent before trying to retrieve it.